### ISSUE TITLE
typescript-compatibility doesn't work with typescript-react-apollo's withHOC: false

### ISSUE THREAD

**Issue by jrr** [dotansimha/graphql-code-generator#1876]
**Describe the bug**
Using `typescript-compatibility` and `typescript-react-apollo` with `withHOC: false`:

```yaml
generates:
  types.gen.tsx:
    config:
      noGraphqlTag: false
      strict: true
    schema:
      - "schema.graphql"
    documents:
      - "documents/*.graphql"
    plugins:
      - typescript:
      - typescript-operations
      - typescript-react-apollo:
          withComponent: false
          withHOC: false
      - typescript-compatibility
```

I get a generated line like this:
`export type Props = EmployeeCreateWidgetsProps;`, with a squiggle under the right side:

```
types.gen.tsx:90:23 - error TS2304: Cannot find name 'EmployeeCreateWidgetsProps'.

90   export type Props = EmployeeCreateWidgetsProps;
                         ~~~~~~~~~~~~~~~~~~~~~~~~~~
```
**To Reproduce**

clone https://github.com/jrr/gql-codegen-issue, yarn install and `yarn run gql-gen -o`, then open types.gen.tsx.

Using latest stable packages.

**Comment by dotansimha** [comment]
@ardatan Maybe it's related to your recent change in `typescript-react-apollo`? (with the `Props`)

**Comment by dotansimha** [comment]
Hi @jrr , fixed in: https://github.com/dotansimha/graphql-code-generator/pull/1889

**Comment by dotansimha** [comment]
Available in `1.2.0`.

**Comment by xiaoyu-tamu** [comment]
it happens again.

    "@graphql-codegen/cli": "^1.4.0",
    "@graphql-codegen/fragment-matcher": "1.4.0",
    "@graphql-codegen/introspection": "1.4.0",
    "@graphql-codegen/typescript": "1.4.0",
    "@graphql-codegen/typescript-compatibility": "1.4.0",
    "@graphql-codegen/typescript-operations": "1.4.0",
    "@graphql-codegen/typescript-react-apollo": "1.4.0",
    "@graphql-codegen/typescript-resolvers": "1.4.0",

**Comment by dotansimha** [comment]
@xiaoyu-tamu can you please create a reproduction for this? I couldn't reproduce it in a test: https://github.com/dotansimha/graphql-code-generator/commit/2369c1d1e4c095e19bc67527adfd8f6b6e8e37cb

**Comment by xiaoyu-tamu** [comment]
@dotansimha try this one 
https://github.com/xiaoyu-tamu/gql-generator-issue

![image](https://user-images.githubusercontent.com/33362998/62421234-3beec780-b664-11e9-9b96-3f5d09e75800.png)

**Comment by dotansimha** [comment]
I think I fixed it in: https://github.com/dotansimha/graphql-code-generator/pull/2285 
@xiaoyu-tamu can you please try it? `1.4.1-alpha-6a24d831.80`

**Comment by xiaoyu-tamu** [comment]
I can confirm it fixed in ^1.4.1-alpha-ffc76e57.81

**Comment by dotansimha** [comment]
Fixed in 1.5.0 🎉
